.github/workflows/build-wheels.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Get new version and build number
       run: |
         pip install python-semantic-release
-        latest_tag=$(git describe --tags --abbrev=0)
+        latest_tag=$(cat VERSION)
```

Comment:
I was having trouble getting tags to work because I would tag commits in a PR, but they ended up not showing up in the stage branch when I merged the PR. (the commits in PR are different from those in stage) I haven't written a workflow that tags commits that are added to stage, so this is my workaround for now.

Reply:
Could you share a workflow where you tag release candidates? That would help me a lot

Reply:
Tools org normally wait to tag in master/main. Here is how asadm does it in make https://github.com/aerospike/aerospike-admin/blob/master/makefile#L41 .